source/usage-examples/find.txt

```diff
@@ -4,13 +4,15 @@ Find Multiple Documents
 
 .. default-domain:: mongodb
 
-You can query for multiple documents in a collection with
-``collection.find()``. The ``find()`` method uses a query document that you
-provide to match the subset of the documents in the collection that match the
-query. If you don't provide a query document (or if you provide an empty
-document), MongoDB returns all documents in the collection. For more
-information on querying MongoDB, see our
-:doc:`documentation on query documents </fundamentals/crud/query-document>`.
+You can query for multiple documents in a collection with the
+``collection.find()`` method. The ``find()`` method uses a query
+document that contains one or more query operators to determine the
+documents to include in the result set. If you don't provide a query
+document or if you provide an empty document, MongoDB returns all
+documents in the collection. For more information on how to 
+specify query operators in a query document, see our
```

Comment:
```suggestion
use query documents and query operators, see our
```
Nit: I think this rephrasing more accurately describes the linked page (more about query documents than query operators). 

Applies to both instances.